test/units/testsuite-19.sh

```diff
@@ -3,6 +3,14 @@
 set -eux
 set -o pipefail
 
+. /etc/os-release
+# FIXME: This test fails on opensuse with the following error and others:
+# Apr 25 10:24:04 H (cat)[910]: device-mapper: create ioctl on ... failed: Device or resource busy
+if [[ "$ID" =~ "opensuse" ]]; then
+    echo Skipping due to known unexpected behaviour in OpenSUSE kernels | tee /skipped >&2
```

Comment:
No need to pipe this to stderr, since we run with `set -x`.

```suggestion
    echo "Skipping due to known unexpected behaviour in OpenSUSE kernels" >>/skipped
```